frontend/src/app/pages/account/account.ts

```diff
@@ -54,14 +59,36 @@ export class AccountPage implements OnInit {
         modalRef.componentInstance.user = this.user;
         modalRef.result.then((result) => {
             if(result && !result.success) {
-                this.createLobbyError = 'Error editing user';
+                this.editInfoError = 'Error editing user';
             } else {
-                console.log('Result: ', result);
                 this.user = result.data;
+                this.accountUpdateSuccess = "Successfully updated account.";
+                setTimeout(() => {
+                    this.accountUpdateSuccess = null;
+                }, 5000);
             }
         }, (reason) => {
 
         });
     }
 
+
+    //experimental code TODO: Add relevant info so it doesn't crash everything
```

Comment:
😂

Reply:
haha I think that was Kyle's comment! Classic 😂